Crash in the threadExample project.

```java
 Caused by: java.lang.IllegalStateException: Changing Realm data can only be done from inside a transaction.
                                                                               at io.realm.internal.Table.throwImmutable(Table.java:1375)
                                                                               at io.realm.internal.Table.checkImmutable(Table.java:1030)
                                                                               at io.realm.internal.Table.clear(Table.java:320)
                                                                               at io.realm.Realm.delete(Realm.java:1265)
                                                                               at io.realm.examples.threads.PassingObjectsFragment.onDestroy(PassingObjectsFragment.java:116)
```
```java
@Override
    public void onDestroy() {
        super.onDestroy();
        // Clear out all Person instances.
        realm.delete(Person.class);
        realm.close();
    }
```
According to the document, `delete` need to put in a transaction.

--- a/examples/threadExample/src/main/java/io/realm/examples/threads/PassingObjectsFragment.java
+++ b/examples/threadExample/src/main/java/io/realm/examples/threads/PassingObjectsFragment.java
@@ -112,8 +112,13 @@ public class PassingObjectsFragment extends Fragment {
     @Override
     public void onDestroy() {
         super.onDestroy();
+        realm.executeTransaction(new Realm.Transaction() {
+            @Override
+            public void execute(Realm realm) {
+                realm.delete(Person.class);
+            }
+        });
         // Clear out all Person instances.
-        realm.delete(Person.class);
         realm.close();
     }
 }

